src/views/Search.vue

```diff
@@ -220,6 +241,10 @@ export default {
         clear: both;
     }
 
+    .topMargin{
```

Comment:
If you aren't familiar with BS4's spacing utility classes, you should be. Sometimes there is a class already available for things like this. So class `mt-2` or `mt-1` might fit your needs (unless 20px is exactly what you need). See https://getbootstrap.com/docs/4.1/utilities/spacing/